iob_soc.py

```diff
@@ -374,11 +375,14 @@ def _custom_setup(cls):
         # Add the following arguments:
         # "INIT_MEM": if should setup with init_mem or not
         # "USE_EXTMEM": if should setup with extmem or not
+        # "SYN": if should setyp with syn or not
```

Comment:
What is different sbou the syn setup?

Reply:
the syn setup is the argument that is passed that is then used during the python setup in order for the python script to understand that its building a syn buildir. then it will not copy certain sources to the buildir.